pkg/store/redis.go

```diff
@@ -22,6 +22,9 @@ type RedisStore struct {
 
 	// The maximum number of retry under race conditions.
 	MaxRetry int
+
+	// Redis connection stub for tests
+	mockGetConnection func() redis.Conn
```

Comment:
This really doesn't feel right... having a mocking method on a struct.
Probably better to have an `TestingStore` where you can mock the `GetConnection` method